projects/packages/my-jetpack/src/products/class-module-product.php

```diff
@@ -86,7 +86,7 @@ public static function is_module_active() {
 	 */
 	public static function get_status() {
 		$status = parent::get_status();
-		if ( 'active' === $status && ! static::is_module_active() ) {
+		if ( 'inactive' === $status && ! static::is_module_active() ) {
```

Comment:
`'active' === $status` does not seem correct here, and it doesn't work properly that way. I don't see how the status could be 'active' and the module be **not** active.
Anyway, I corrected this to `'inactive' === $status` which works correctly, as expected now.